aa_forum/templates/aa_forum/partials/forum/board/board-index.html

```diff
@@ -31,7 +31,7 @@
     </div>
 </div>
 
-{% include "aa_forum/partials/forum/board/new-topic-button.html" %}
+{% include "aa_forum/partials/forum/board/new-topic-button.html" with num=2 %}
```

Comment:
Arguments that are added to an ID string should be descriptive.

position=above-topic-list / below-topic-list

Reply:
fixed